web/src/main/webapp/dataViews/tableView.ts

```diff
@@ -744,6 +720,80 @@ export class TableView extends TSViewBase implements IScrollTarget, OnNextK {
                         },
                         help: "Extract a value associated with a specific key."
                     }, !this.isPrivate());
+            this.contextMenu.insertSubMenu( "Charts...",
```

Comment:
Instead of specifying here the text to search for, it would be better for the method addExpandableMenuItem to return a more useful object which has methods to add new items.